### DISCUSSION TITLE
Request interception for cache update

### DISCUSSION THREAD

**Discussion by hexwit** [urql-graphql/urql#1088] [Q&A] [answered]
Hello,

I have normalized cache in the application. 
I have a list of items on the page and I item details opens in modal window on the same page. 
Now I want on item update, add, delete update the items list. 

Is there any way to achieve this behavior with urql, on its level?

**Comment by JoviDeCroock** [comment, marked as answer]
We have documentation about custom updaters in our normalized cache [here](https://formidable.com/open-source/urql/docs/graphcache/custom-updates/)
